packager/packager.cc

```diff
@@ -599,13 +689,16 @@ ast::ParsedFile rewritePackage(core::Context ctx, ast::ParsedFile file, const Pa
     return file;
 }
 
-ast::ParsedFile rewritePackagedFile(core::Context ctx, ast::ParsedFile file, core::NameRef packageMangledName) {
+ast::ParsedFile rewritePackagedFile(core::MutableContext ctx, ast::ParsedFile file, core::NameRef packageMangledName,
+                                    const PackageInfo *pkg) {
     if (ast::isa_tree<ast::EmptyTree>(file.tree)) {
         // Nothing to wrap. This occurs when a file is marked typed: Ignore.
         return file;
     }
 
     auto &rootKlass = ast::cast_tree_nonnull<ast::ClassDef>(file.tree);
+    EnforcePackagePrefix enforcePrefix(pkg);
+    file.tree = ast::TreeMap::apply(ctx, enforcePrefix, move(file.tree));
```

Comment:
Sorbet has two AST traversals:

* `ast::TreeMap` traverses the entire AST, including method bodies.
* `ast::ShallowMap` traverses just the outside definitions and doesn't traverse method bodies.

The latter is a bit faster. I think you don't need to traverse method bodies here.

```suggestion
    file.tree = ast::ShallowMap::apply(ctx, enforcePrefix, move(file.tree));
```